Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts

```diff
@@ -285,34 +285,41 @@ function expandSymbolsPaths(symbolsType: string, pattern: string, continueOnErro
         // multiple dSYM folder paths for Apple application.
         let dsymPaths = utils.resolvePaths(pattern, continueOnError, packParentFolder);
 
-        dsymPaths.forEach(dsymFolder => {
-            if (dsymFolder) {
-                let folderPath = utils.checkAndFixFilePath(dsymFolder, continueOnError);
-                // The path can be null if continueIfSymbolsNotFound is true and the folder does not exist.
-                if (folderPath) {
-                    symbolsPaths.push(folderPath);
+        // Resolved paths can be null if continueIfSymbolsNotFound is true and the file/folder does not exist.
+        if (dsymPaths) {
+            dsymPaths.forEach(dsymFolder => {
```

Comment:
I prefer "for" over "forEach", "for" is faster too but in this case it might not matter that much.